File: cpp/array/stack.cpp
#include <map>
#include <iostream>

using std::string;
using std::cout;
using std::endl;




int main() {
    std::map<string, string> count;
    string list[] = {"abc", "def"};
    for (auto s: list) {
        cout << s << endl;
    }
    return 0;
}

File: csharp/array/Stack.cs
using System;
using System.Collections;

// Implementing a stack is trivial using a dynamic array
// (which we implemented earlier).
public class Stack
{
    ArrayList stack = new();

    public Stack() {}

    public void Push(int n)
    {
        stack.Add(n);
    }

    public int Pop()
    {
        if (Size() > 0)
        {
            int ele = (int)stack[stack.Count - 1];
            stack.RemoveAt(stack.Count - 1);
            return ele; 
        }
        return -1;
    }

    public int Size()
    {
        return stack.Count;
    }
}